Configuration/AlCa/python/autoCond.py

```diff
@@ -68,15 +68,15 @@
     # GlobalTag for MC production with perfectly aligned and calibrated detector for Phase1 2021
     'phase1_2021_design'           : '121X_mcRun3_2021_design_v4',
     # GlobalTag for MC production with realistic conditions for Phase1 2021
-    'phase1_2021_realistic'        : '121X_mcRun3_2021_realistic_v5', 
+    'phase1_2021_realistic'        : '121X_mcRun3_2021_realistic_Candidate_2021_09_25_15_31_23',
```

Comment:
@cms-sw/alca-l2 please notice that I could not produce a new candidate GT without the record `DTRecoUncertaintiesRcd` (as done in PR https://github.com/cms-sw/cmssw/pull/35386) as it seems that the change was not yet propagated to the Queue.

Reply:
Actually not relevant after rebase.